sched/scheduler/scheduler_test.go

```diff
@@ -1,6 +1,6 @@
 package scheduler
 
-import (
+/*import (
```

Comment:
I'm guessing you commented this out so you could run other tests (which is a silliness of go's tooling that you had to) and that you're in the process or reenabling it.